src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/TableMappingTest.java

```diff
@@ -215,7 +215,7 @@ void getIndexPrimaryKeyFieldMappings() {
     @Test
     void validateVirtualPhysicalCompatibility_missingVirtualHk() {
         assertException((TestFunction<NullPointerException>) () ->
-                        sut.validateCompatiblePrimaryKey(new PrimaryKey(null, S), null),
+                        sut.validateCompatiblePrimaryKey(new PrimaryKey(null, S), new PrimaryKey("hk", N)),
```

Comment:
why was this missing to begin with?

Reply:
it's validating the first argument, the 2nd argument was ignored in this particular test